GameData/RealismOverhaul/Engine_Configs/S2_253_Config.cfg

```diff
@@ -1,18 +1,93 @@
-//	
+
+//	==================================================
+//	S2.253 and derivatives
+//
+//	Manufacturer: Makeyev Rocket Design Bureau
+//
+//	=================================================================================
+//	S2.253 (8D511)
+//	R-11 (Scud-A, 1953)
 //
+//	Dry Mass: 300 kg
+//	Thrust (SL): 81.4 kN
+//	Thrust (Vac): 94.8 kN
+//	ISP: 219 SL / 251 Vac
+//	Burn Time: 90
+//	Chamber Pressure: ???
+//	Propellant: AK20 / Kerosene
+//	Prop Ratio: 3.07
+//	Engine Nozzle: ???
+//	Nozzle Exit Area: ???
+//	Nozzle Ratio: ???
+//	Ignitions: 1
+//	=================================================================================
+//	S3.42T
+//	R-11MU, R-17 prototype (1958)
+//
+//	Dry Mass: 160 kg
+//	Thrust (SL): 111.7 kN
+//	Thrust (Vac): 127.5 kN
+//	ISP: 226 SL / 258 Vac
+//	Burn Time: 65
+//	Chamber Pressure: ??? MPa
+//	Propellant: AK27 / Kerosene
+//	Prop Ratio: 3.55	(calculated from fuel loading of R-17)
+//	Engine Nozzle: ???
+//	Nozzle Exit Area: ???
+//	Nozzle Ratio: ???
+//	Ignitions: 1
+//	=================================================================================
+//	S5.2
+//	R-17 (Scud-B, Scud-C)
+//
+//	Dry Mass: 160 kg
+//	Thrust (SL): 132.1 kN
+//	Thrust (Vac): 146.3 kN
+//	ISP: 226 SL / 258 Vac
+//	Burn Time: 75
+//	Chamber Pressure: ??? MPa
+//	Propellant: AK27 / Kerosene
+//	Prop Ratio: 3.55
+//	Engine Nozzle: ???
+//	Nozzle Exit Area: ???
+//	Nozzle Ratio: 8.2
+//	Ignitions: 1
+//	=================================================================================
+//	Isayev-R17
+//	R-17MU (Scud-D)
+//
+//	Dry Mass: 160 kg
+//	Thrust (SL): 149.9 kN
+//	Thrust (Vac): 165.5 kN
+//	ISP: 240 SL / 265 Vac
+//	Burn Time: 90
+//	Chamber Pressure: ??? MPa
+//	Propellant: AK27 / UDMH
+//	Prop Ratio: 2.32	//8.5 m2 UDMH, 10.46 m2 AK27, 1.23 volume
+//	Engine Nozzle: ???
+//	Nozzle Exit Area: ???
+//	Nozzle Ratio: 9.6
+//	Ignitions: 1
+
+//	Sources:
+//	b14643 - The Soviet "Scud" missile family:													http://www.b14643.de/Spacerockets/Specials/Scud/
+
+
+//	Used by:
+
+//	Notes:
+
+//	==================================================
 
 @PART[*]:HAS[#engineType[S2.253]]:FOR[RealismOverhaulEngines]
 {
 	%category = Engine
 	%title = S2.253 Engine
-	%manufacturer = 
-	%description = The S2.253 engine was developed for use in the R-11 Zemlya ballistic missile, based on the Wasserfall engine using Kerosene and Nitric Acid as propellants.
+	%manufacturer = OKB Isayev
+	%description = The S2.253 engine was developed for use in the R-11 Zemlya ballistic missile and sounding rocket, based on the German Wasserfall engine. Better known by its western designation, Scud, later versions where exported extensivley to many countries, eventually forming the basis of the Iranian and North Korean space programs.
 
 	@MODULE[ModuleEngines,*]
```

Comment:
Not resolved.  Line 89: `@MODULE[ModuleEngines,*]` has a `,` inside of the `[]`.

Reply:
"later versions where" -> "later versions were"